scripts/flow/common.js

```diff
@@ -36,5 +36,5 @@ export async function processFlowInfo(flowInfo) {
     (temp_macro.flags[MODULE_ID] ||= {}).flowInfo = flowInfo;
     temp_macro.ownership.default = CONST.DOCUMENT_OWNERSHIP_LEVELS.OWNER;
 
-    temp_macro.execute({ actor: sourceToken.actor, token: sourceToken });
```

Comment:
Unrelated tiny non-fix for a strange crash report seen in the Discord. This should push the error deeper and hopefully produce a more interesting log the next time someone hits the maybe-bug